genesis-processing/db/args.py

```diff
@@ -0,0 +1,78 @@
+from argparse import ArgumentParser
```

Comment:
Honestly think that I would put all of this in the `main.py` probably unless it is reused?